pkg/plugin/vm_restore_item_action.go

```diff
@@ -63,6 +63,10 @@ func (p *VMRestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (
 		return nil, errors.WithStack(err)
 	}
 
+	for i := 0; i < len(vm.Spec.Template.Spec.Domain.Devices.Interfaces); i++ {
+		vm.Spec.Template.Spec.Domain.Devices.Interfaces[i].MacAddress = ""
```

Comment:
What about users that want to preserve the mac address?

I think a better solution would be to add an annotation to the VM like `backup.kubevirt.io/clearMacAddress: true` and if that's set, the plugin will clear the mac

Reply:
right, added annotation check